types/react/index.d.ts

```diff
@@ -741,14 +968,14 @@ declare namespace React {
          */
         displayName?: string | undefined;
         /**
-         * defaultProps are not supported on components passed to forwardRef.
+         * defaultProps are not supported on render functions passed to forwardRef.
```

Comment:
@eps1lon Here are the changes you requested - good note on the wording here.

Reply:
See feedback here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68410#discussion_r1473960546